guides/v3.2.0/object-model/computed-properties-and-aggregate-data.md

```diff
@@ -25,7 +25,7 @@ export default Component.extend({
   },
 
   titles: computed('todos.[]', function() {
-    let todos = this.get('todos');
+    let todos = this.todos;
```

Comment:
Could probably just inline `this.todos` in the following line, unless you think there's extra clarity separating it out.